app/src/service/ocr/document/adpEarningsStatement.ts

```diff
@@ -0,0 +1,20 @@
+import { createDocumentMatcher } from "@/service/factory";
+
+export const adpEarningsStatement = createDocumentMatcher(
+  "ADP Earnings Statement",
+  "adpEarningsStatement",
+  {
+    company: /(H GREG NISSAN DELRAY LLC)/,
```

Comment:
I realize that these are currently hard-coded. We can work on defining new regex patterns (and ideally functions) used to extract relevant information from the OCR text results.